recipes/afterburner/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "metoffice-afterburner" %}
+{% set version = "1.3.2b1.post4" %}
+{% set sha256 = "e5603a67d024b6fc0c1142a868f1ae9018511f34de25f3e8b05aec0941617e52" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  string: py36
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - rsync
+  host:
+    - rsync
+    - python
+    - pip
+  run:
+    - python ==3.6.*
```

Comment:
Does this package only work on Python 3.6? If so, that should be handled above in the `skip`:

`skip: True  # [not (linux and py36)]`

Reply:
In principle, the metoffice-afterburner package should work with any version of Python >= 2.7. There are, however, complex interdependencies among some of the prerequisite packages, notably iris, cartopy, dask and numpy. So in an attempt to get the recipe working I thought I'd constrain the version of Iris and friends to those used in our production environment here at work (where Iris was conceived!).

I'll try resetting Python to >= 3.6 since that should be the baseline from here on.